kapitan/refs/secrets/azkms.py

```diff
@@ -38,12 +40,28 @@
 # export AZURE_CLIENT_ID="generated app id"
 # export AZURE_CLIENT_SECRET="random password"
 # export AZURE_TENANT_ID="tenant id"
-# export AZ_VAULT_URL=<vault name>
+# export AZ_VAULT=<vault name>
 # export AZ_KEY_NAME=<name of the key used to ecrypt/decrypt
 # Remember to set the permissions -
 # az keyvault set-policy --name my-key-vault --spn $AZURE_CLIENT_ID --key-permissions get encrypt decrypt
 
 
+class EncryptionAlgorithm(str, Enum):
+    """Encryption algorithms"""
+
+    rsa_oaep = "RSA-OAEP"
+    rsa_oaep_256 = "RSA-OAEP-256"
+    rsa1_5 = "RSA1_5"
+
+    # an old classic
+    @staticmethod
+    def value_of(value):
+        for m, mm in EncryptionAlgorithm.__members__.items():
```

Comment:
What is `m` and what is `mm` in this context? 
One of them is algorithm members probably. But it helps readability a lot if we use longer-named variables here.

Reply:
for _attribute_name, _ in EncryptionAlgorithm.__members__.items():